Workspace/apps/admin-dashboard/test/functional/charts.js

```diff
@@ -0,0 +1,62 @@
+describe("Checking charts", function() {
+
+    var Dash = {
+        chartsAreaPanel : function () {
+            return ST.component('chartsareapanel');
+        },
+
+        chartsPolarPanel : function () {
+            return ST.component('chartspolarpanel');
+        },
+
+        chartsStackedPanel : function () {
+            return ST.component('chartsstackedpanel');
+        },
+
+        chartsBarPanel : function () {
+            return ST.component('chartsbarpanel');
+        },
+
+        chartsGaugePanel : function () {
+            return ST.component('chartsgaugepanel');
+        },
+
+        chartsPie3dPanel : function () {
+            return ST.component('chartspie3dpanel');
+        },
+
+        chartsPage : function () {
+            return ST.component('charts');
+        }
+    };
+        
+    beforeEach(function(){
+        Admin.app.redirectTo("#charts"); // make sure you are on dashboard homepage
+    });
+
+    it('Take a screenshot', function (next) {
```

Comment:
Let's use `done` for the name of the parameter instead of `next` as it matches Jasmine docs